.github/workflows/rust-static-lib.yml

```diff
@@ -0,0 +1,73 @@
+name: binding-rust-static-lib
```

Comment:
Hi @apepkuss 
IMO, we should collect `rust*` workflows into one file. We can reuse many similar parts instead of creating more and more rust-related workflow files, and this will increase the maintenance cost.

Reply:
This new workflow can be seen as a temp workflow as it could be changed (maybe a lot). Therefore, use a separate workflow for now. But, I agree with the point to move all `rust*` workflows into one file. I believe creating another PR to do this job would be better. Thanks!